docker-compose.yml

```diff
@@ -51,17 +51,22 @@ services:
 
   clickhouse:
     image: "yandex/clickhouse-server:latest"
-
-  # This is necessary for doing live React dev for the new board, but isn't needed after doing a build
-  # if the nginx config is changed to not redirect /beta/ to port 3000:
+  webserver:
```

Comment:
Put a comment here explaining why this is necessary so @kestasjk doesn't remove it again.

Reply:
I put it back to where it was